05_select_random_elements_from_array.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Implementation for RubyMonk Primer Problem 5 Select random elements from an array
+# Problem Statement:
+# Create a method 'random_select' which, when given an array of elements (array) and
+# a number (n), returns n randomly selected elements from that array.
+# Example:
+# Given an array [1, 2, 3, 4, 5] and 2 should return two random numbers from the given
+# array. (Note: those two random numbers may be the same number.) The method should
+# return those random values in a new array.
+# Calling the method twice should ideally return different sets of values
+# (though it may not).
+# The exercise is almost to write your own Array#sample, so it's been disallowed.
+class SelectRandomElementsFromArray
+  # rubocop:disable Naming/MethodParameterName
+  def random_select(array, n)
+    # your code here
+    random_elements = []
+    n.times do
+      random_elements << array[rand(array.count)]
```

Comment:
I'd probably do this as something like
```ruby
n.times do
  random_elements << array.sample
end
```

Another alternative might be to do something like
```ruby
array.shuffle.slice(0, n)
```
which will randomize the array, then grab the first `n` elements

Reply:
LOL.  The problem instructions specifically said NOt to use `#sample` (and there's a test to enforce this) but the other suggestions are gold.

Reply:
Oh I missed the bit on `sample` 🤦🏻

Reply:
:white_check_mark: